storage-proofs-post/src/fallback/utils.rs

```diff
@@ -0,0 +1,19 @@
+use storage_proofs_core::api_version::ApiVersion;
+
+/// Selects the challenge index used to determine the leaf challenge for PoSt
+pub fn get_challenge_index(
+    api_version: ApiVersion,
+    sector: usize,
+    sector_chunk_index: usize,
+    num_sectors_per_chunk: usize,
+    challenge_count: usize,
+    challenge_index: usize,
```

Comment:
I would maybe change these argument names to something like:

```rust
fn get_challenge_index(
    api_version,
    sector_chunk_index,
    num_sectors_per_chunk,
    sector_index_in_chunk.
    challenge_count_per_sector,
    challenge_index,
)
```
or

```rust
fn get_challenge_index(
    api_version,
    sector_index,
    challenge_count_per_sector,
    challenge_index,
)
```
where the caller of `get_challenge_index` passes in the sector index in the PoSt proof `sector_index = chunk_index * num_sectors_per_chunk + sector_index_in_chunk`.

Reply:
Nice, yeah I think something like that will be cleaner :+1: